--- mumble-1.2.10.orig/src/murmur/Meta.cpp	2015-12-05 14:58:43.370750681 -0600
+++ mumble-1.2.10/src/murmur/Meta.cpp	2015-12-05 15:07:53.375617170 -0600
@@ -253,6 +253,10 @@
 		}
 		if (hasipv4)
 			qlBind << QHostAddress(QHostAddress::Any);
+
+                if (!hasipv4 && !hasipv6)
+                        qFatal("Can not bind any address.");
+
 	}
 
 	qsPassword = typeCheckedFromSettings("serverpassword", qsPassword);
