--- mumble-1.1.8/compiler.pri.orig	2010-03-23 14:52:21.241679313 +0000
+++ mumble-1.1.8/compiler.pri	2010-03-23 14:52:38.983679366 +0000
@@ -49,7 +49,7 @@
 	QMAKE_CFLAGS *= -Wshadow -Wconversion -Wsign-compare -fvisibility=hidden
 	QMAKE_CXXFLAGS *= -Wshadow -Woverloaded-virtual -Wold-style-cast -Wconversion -Wsign-compare -fvisibility=hidden
 	!macx {
-		QMAKE_LFLAGS *= -Wl,--as-needed
+		QMAKE_LFLAGS *= -Wl,--as-needed -Wl,--add-needed
 	}
 
 	CONFIG(optgen) {
