diff -up mumble-1.2.19/compiler.pri.cflags mumble-1.2.19/compiler.pri
--- mumble-1.2.19/compiler.pri.cflags	2017-01-27 00:48:33.000000000 -0600
+++ mumble-1.2.19/compiler.pri	2018-03-21 08:50:33.132916349 -0500
@@ -85,8 +85,8 @@ win32 {
 
 unix {
 	DEFINES *= RESTRICT=__restrict__
-	QMAKE_CFLAGS *= -Wfatal-errors -fvisibility=hidden
-	QMAKE_CXXFLAGS *= -Wfatal-errors -fvisibility=hidden
+	QMAKE_CFLAGS *= -fvisibility=hidden
+	QMAKE_CXXFLAGS *= -fvisibility=hidden
 	!CONFIG(quiet-build-log) {
 		QMAKE_CFLAGS *= -Wshadow -Wconversion -Wsign-compare
 		QMAKE_CXXFLAGS *= -Wshadow -Woverloaded-virtual -Wold-style-cast -Wconversion -Wsign-compare
